tgui/packages/tgui/interfaces/PlayerPreferences.js

```diff
@@ -0,0 +1,1282 @@
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={medicalDesc}
+              onChange={(e, value) => setMedicalDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+        <Grid.Column>
+          <Section
+            title="Employment Records"
+            buttons={
+              <Box>
+                <Button
+                  icon="save"
+                  disabled={employmentDesc === data.gen_record}
+                  onClick={() => act('gen_record', { employmentDesc })}>
+                  Save
+                </Button>
+                <Button
+                  icon="times"
+                  onClick={() => setEmploymentDesc(data.gen_record)}>
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={employmentDesc}
+              onChange={(e, value) => setEmploymentDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+      </Grid>
+      <Grid>
+        <Grid.Column>
+          <Section
+            title="Security Records"
+            buttons={
+              <Box>
+                <Button
+                  icon="save"
+                  disabled={securityDesc === data.sec_record}
+                  onClick={() => act('sec_record', { securityDesc })}>
+                  Save
+                </Button>
+                <Button
+                  icon="times"
+                  onClick={() => setSecurityDesc(data.sec_record)}>
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={securityDesc}
+              onChange={(e, value) => setSecurityDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+        <Grid.Column>
+          <Section
+            title="Exploit Records"
+            buttons={
+              <Box>
+                <Button
+                  icon="save"
+                  disabled={exploitsDesc === data.exploit_record}
+                  onClick={() => act('exploit_record', { exploitsDesc })}>
+                  Save
+                </Button>
+                <Button
+                  icon="times"
+                  onClick={() => setExploitsDesc(data.exploit_record)}>
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={exploitsDesc}
+              onChange={(e, value) => setExploitsDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+      </Grid>
+    </Section>
+  );
+};   
+
+const CharacterCustomization = (props, context) => {
+  const { act, data, config } = useBackend(context);
+
+  const rgbToHex = (red, green, blue) => {
```

Comment:
this might have something similar already, if not i'll search where you can put this, its a good helper

Reply:
Color class probably.